### PR TITLE
nothing loads before config

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

File: src/libs/config.js [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
 
 export const configOverridesStore = Utils.atom()
 Utils.syncAtomToSessionStorage(configOverridesStore, 'config-overrides')
 // Values in this store will override config settings. This can be used from the console for testing.
 window.configOverridesStore = configOverridesStore
 
 export const getConfig = () => {
-  console.assert(loadedConfig, 'Called getConfig before iniitialization')
-  return _.merge(loadedConfig, configOverridesStore.get())
+  console.assert(loadedConfigStore.current, 'Called getConfig before iniitialization')
```
Comment (katevoss) [inline src/libs/config.js]:
```suggestion
  console.assert(loadedConfigStore.current, 'Called getConfig before initialization')
```

**Review by katevoss — APPROVED** [review]
I poked around and everything looks fine, let me know if there's something specific you want me to test.

But please oh please fix the spelling error, I've been waiting for someone to be in this part of the code to fix it! 🙏

File: src/configStore.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const loadedConfigStore = { config: undefined }
```
Comment (panentheos) [inline src/configStore.js]: This should say `current`

**Review by panentheos — APPROVED** [review]
Looks good, thanks!

**Pull request merged**